internal/server/iifo.go

```diff
@@ -1,12 +1,10 @@
 package server
 
-import "github.com/SOMAS2020/SOMAS2020/internal/common"
-
 // runIIFO : IIFO makes recommendations about the optimal (and fairest) contributions this term.
 // to mitigate common risk dilemma.
-func (s *SOMASServer) runIIFO() ([]common.Action, error) {

```

Comment:
I assume this is in line with what we said this afternoon about not caring about actions for the time being - if we need it we'll put it in as discussed in #56?